### PR TITLE
fixed bug that caused ima plugin not to work in IE10

### PR DESCRIPTION
**Pull request by aronallen**
The plugin wasn't working as described in this googleads/videojs-ima#44
I think I would like to share my changes back to the main repo.

### CODE REVIEW

File: src/videojs.ima.js [hunk 1]
```diff
@@ -17,7 +17,22 @@
...
+    });
+  } else if (typeof exports === 'object') {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory(require("video.js"));
+  } else {
+    root['videojs-ima'] = factory(videojs);
+  }
+}(this, function(vjs) {
```
Comment (shawnbuso) [inline src/videojs.ima.js]: Could you expand on the changes above? Are they required for the IE change to work? If not, could you pull them out and (if required) submit them as a separate pull request?

**Comment by shawnbuso** [comment]
Thanks for the submission! It looks like you still need to sign our CLA in order for us to accept your code. Please see https://github.com/googleads/videojs-ima/blob/master/CONTRIBUTING.md for full info on how to sign so we can accept.

File: src/videojs.ima.js (hunk 1 above)
Comment (aronallen) [inline src/videojs.ima.js]: the umd fix is just for me, I should perhaps had put that in a separate branch.
the only thing I want to post back is  the changes i did with append child.

**Comment by aronallen** [comment]
@shawnbuso it appears you already have a pull request pending that fixes the same thing #126

**Comment by shawnbuso** [comment]
Closing for lack of activity. If you'd like this to be merged please sign the CLA linked above and re-open. Thanks!
